### PR TITLE
backend koofr: remove trailing bracket - fixes #7600

### LINKED ISSUE #7600: Docs include trailing bracket into URL
https://rclone.org/koofr/#standard-options have this section:

grafik

As you can see, the trailing bracket `)` is included in the link of the URL, making it not working when clicking it:

`https://app.koofr.net/app/admin/preferences/password)`

The docs seem to come from https://github.com/rclone/rclone/blob/master/docs/content/koofr.md. This file doesn't have the problem:

grafik

I assume that this is some issue with the conversion of Markdown to the docs HTML. Possibly also an upstream issue.

### PR DESCRIPTION
**Pull request by KDreynolds**
#### Was the change discussed in an issue or in the forum before?

#7600 

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: backend/koofr/koofr.go [hunk 1]
```diff
@@ -67,13 +67,13 @@ func init() {
...
 			Name:       "password",
-			Help:       "Your password for rclone (generate one at https://app.koofr.net/app/admin/preferences/password).",
+			Help:       "Your password for rclone (generate one at https://app.koofr.net/app/admin/preferences/password.",
 			Provider:   "koofr",
 			IsPassword: true,
 			Required:   true,
 		}, {
 			Name:       "password",
-			Help:       "Your password for rclone (generate one at https://storage.rcs-rds.ro/app/admin/preferences/password).",
+			Help:       "Your password for rclone (generate one at https://storage.rcs-rds.ro/app/admin/preferences/password.",
```
Comment (albertony) [inline backend/koofr/koofr.go:76]: Both of the changes only removes the ending paren `)`, not the initial `(`?

**Comment by KDreynolds** [comment]
ok not sure why linting is failing, tried go fmt on changed file to no avail. any help is appreciated.

**Comment by albertony** [comment]
> ok not sure why linting is failing, tried go fmt on changed file to no avail. any help is appreciated.

Its fine, unrelated to your changes.

File: backend/koofr/koofr.go (hunk 1 above)
Comment (KDreynolds) [reply backend/koofr/koofr.go:76]: I am not a smart man...

Comment (KDreynolds) [reply backend/koofr/koofr.go:76]: fixing

**Review by ncw — APPROVED** [review]
Thank you :-)

**Pull request merged**